src/helio.jl

```diff
@@ -0,0 +1,139 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _helio(jd::T, num::Integer, radian::Bool) where {T<:AbstractFloat}
```

Comment:
What about using the planet name instead of a numerical index?  What do you think?